task9.js

```diff
@@ -0,0 +1,22 @@
+// Write a function printEvenNumbers(limit) that prints all even numbers from 1 to the given limit using a 'for' loop.
+
+const printEvenNumbers = (limit) => {
+  if (
+    typeof limit !== "number" ||
+    Number.isNaN(limit) ||
+    !Number.isInteger(limit) ||
+    limit < 1
+  ) {
+    throw new Error(
+      "Provided value must be a valid integer, that starts from 1"
+    );
+  }
+
+  for (let i = 1; i <= limit; i++) {
```

Comment:
You can start the loop at `2` and increment by` 2` to directly print even numbers, avoiding the need for the `if` statement inside the loop.